File: QTD.SPA/src/app/_DtoModels/DIFSurvey/DIFSurveyEmployeeResponseModel.ts
export class DIFSurveyEmployeeResponseModel  {
    dIFSurveyTaskId: string;
    difficulty: number;
    importance: number;
    frequency: number;
    nA: boolean;
    comments: string;

    constructor(taskId: string, diff: number, imp: number, freq: number, na: boolean, comment: string){
        this.dIFSurveyTaskId = taskId;
        this.difficulty = diff;
        this.importance = imp;
        this.frequency = freq;
        this.nA = na;
        this.comments = comment;
    }
  }

File: QTD2.Infrastructure/Model/DIFSurvey_Employee/DIFSurveyEmployeeResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.DIFSurvey
{
    public class DIFSurveyEmployeeResponseModel
    {
        public int DIFSurveyTaskId { get; set; }
        public float? Difficulty { get; set; }
        public float? Importance { get; set; }
        public float? Frequency { get; set; }
        public bool NA { get; set; }
        public string? Comments { get; set; }
        public DIFSurveyEmployeeResponseModel()
        {

        }
    }

}
